rero_ils/modules/notifications/api.py

```diff
@@ -176,6 +176,12 @@ def organisation_pid(self):
         self.init_loan()
         return self.transaction_location.organisation_pid
 
+    @property
+    def organisation(self):
+        """Shortcut for organisation of the notification."""
+        from ..organisations.api import Organisation
```

Comment:
I think you can move this import to the top of the file. if it is not possible because a circular importing, it is fine here.

Reply:
I just used the same pattern than `def item(self)` few lines below ;-)